### PR TITLE
feat: add support for AWTK(a graphics library for many kinds of platf…

### PR DESCRIPTION
**Pull request by ufbycd**
## Proposed changes
Add a new driver to the dependencies list to support for [AWTK](https://github.com/zlgopen/awtk), and define a new platform call AWTK.
AWTK apps can run on Windows Linux macOS Android IOS RTOS and so on.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by jlipps** [comment]
We are not going to be merging any more drivers into the 1.x version series. With Appium 2.0 it's not necessary for drivers to be mentioned in the main Appium project code base at all. Is your driver compatible with Appium 2.0?

File: package.json
Comment (mykola-mokhnach) [inline package.json:45]: Please remove the driver from dependencies, but keep the other stuff. The idea is users could add the necessary package(s) themselves if they need it. See, for example, the discussion on https://github.com/appium/appium/pull/15227
We are currently working on Appium 2, which could load drivers dynamically, while Appium 1 would be deprecated anyway.

**Comment by ufbycd** [comment]
> We are not going to be merging any more drivers into the 1.x version series. With Appium 2.0 it's not necessary for drivers to be mentioned in the main Appium project code base at all. Is your driver compatible with Appium 2.0?

This driver is just for 1.x version.

File: package.json
Comment (ufbycd) [reply package.json]: Got it. I've deleted it. 
Looking forward to Appium 2！

**Comment by jlipps** [comment]
all that's required for appium 2.0 compatibility is adding some fields into the package.json, basically!

**Review by mykola-mokhnach — APPROVED** [review]

**Comment by mykola-mokhnach** [comment]
@ufbycd Could you please sign the CLA?
Make sure your local git email address matches to the one used to sign CLA if you've already done it

**Review by jlipps — APPROVED** [review]

**Comment by mykola-mokhnach** [comment]
@ufbycd Any updates on the CLA signing? We could merge this PR immediately after this is done

**Comment by mykola-mokhnach** [comment]
Unfortunately we cannot merge this PR without CLA being signed. Thus closing

**Pull request closed without merging**
